cloud/configs/deployment/scripts/prepare_server

```diff
@@ -32,14 +32,14 @@ if [ $? != 0 ]; then
 fi
 
 # Copy specified envdir from S3
-sudo aws s3 cp s3://endagaweb-deployment/environments/$DEPLOY_ENV /var/opt/endagaweb-envdir --region ap-northeast-1 --recursive
+sudo aws s3 cp s3://endagaweb-deployment-aricent-aricent/environments/$DEPLOY_ENV /var/opt/endagaweb-envdir --region ap-northeast-1 --recursive
```

Comment:
these changes should be parameterised so that the script can figure out (somehow) the correct s3 bucket to pull from. Although I assume that since the AWS actions are all scoped by the account used to perform them that there should be no need to change the bucket name anyway (you don't need to add `aricent-aricent` to the name).